ncm-metaconfig/src/main/metaconfig/nrpe/pan/schema.pan

```diff
@@ -0,0 +1,27 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+
+declaration template metaconfig/nrpe/schema;
+
+include 'pan/types';
+
+@{Configuration options for Nagios Remote Plugin Executor (NRPE)}
+type nrpe_service = {
+    'log_facility'              : string = 'daemon'
+    'pid_file'                  : string = '/var/run/nrpe.pid'
+    'server_port'               : type_port = 5666
+    'server_address'            ? string
```

Comment:
As I understand it, this is the address for nrpe to bind to on start, so either `type_ip` or `type_hostname` (if hostnames are resolved by nrpe) should work here.